charts/refinery/values.yaml

```diff
@@ -104,6 +104,22 @@ resources:
     cpu: 500m
     memory: 500Mi
 
+# liveness probe configuration
+# Ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+livenessProbe:
+  initialDelaySeconds: 10
+  periodSeconds: 3
+  timeoutSeconds: 1
+  failureThreshold: 3
+
+# readiness probe configuration
+# Ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+readinessProbe:
+  initialDelaySeconds: 0
+  periodSeconds: 1
+  timeoutSeconds: 1
+  failureThreshold: 3
```

Comment:
I think we want failure threshold for ready to be 1 -- if it ever responds not ready, it's not ready. I might add like a 3 second initialDelay, though, just so we don't time out the first request.